### PR TITLE
Update ITEM(5900-5999 and other tidbits).tsv

### PR DESCRIPTION
**Pull request by Ekatrina**
---

5999대왕: Emperor vs great king
trial vs ordeal vs test
5904 부서진 르귄가의 인장 조각 Broken Le Guin family's seal piece   (seal and stamp are interchangeable)
5919 Kono Dio da!

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5897,30 +5897,30 @@ ITEM_20150729_005896
...
+ITEM_20150729_005915	Gravekeeper's Torch
+ITEM_20150729_005916	Torch received from Sigis. It is used to cremate bodies.
+ITEM_20150729_005917	Gravekeeper's voodoo  doll
+ITEM_20150729_005918	Gravekeeper's bottled voodoo doll.
 ITEM_20150730_005919	Dio Two-Handed Sword
 ITEM_20150730_005920	Dio Crossbow
-ITEM_20150730_005921	Tresh Spear
-ITEM_20150730_005922	Sestas Pike
-ITEM_20150803_005923	Krausas Mace
+ITEM_20150730_005921	Trash Spear
```
Comment (Sunaries) [inline ITEM.tsv]: Trash -> Thresh

https://github.com/Treeofsavior/EnglishTranslation/wiki/2.3.%20Equipment

File: ITEM.tsv [hunk 2]
```diff
@@ -5897,30 +5897,30 @@ ITEM_20150729_005896
...
+ITEM_20150730_005922	Sestos Pike
```
Comment (Sunaries) [inline ITEM.tsv]: Sestos -> Sestas

https://github.com/Treeofsavior/EnglishTranslation/wiki/2.3.%20Equipment

File: ITEM.tsv [hunk 3]
```diff
@@ -5994,6 +5994,6 @@ ITEM_20150804_005993
 ITEM_20150804_005994	
 ITEM_20150804_005995	
 ITEM_20150804_005996	
-ITEM_20150804_005997	
-ITEM_20150804_005998	
-ITEM_20150804_005999	
+ITEM_20150804_005997	Proof of the emperor's second trial
```
Comment (Sunaries) [inline ITEM.tsv]: Capitalize the e in Emperor.

File: ITEM.tsv [hunk 4]
```diff
@@ -5994,6 +5994,6 @@ ITEM_20150804_005993
...
+ITEM_20150804_005998	Proof of the emperor's third trial
```
Comment (Sunaries) [inline ITEM.tsv]: Capitalize the e in Emperor.

File: ITEM.tsv [hunk 5]
```diff
@@ -5994,6 +5994,6 @@ ITEM_20150804_005993
...
+ITEM_20150804_005999	Proof of the emperor's fourth trial
```
Comment (Sunaries) [inline ITEM.tsv]: Capitalize the e in Emperor.

File: ITEM.tsv [hunk 6]
```diff
@@ -5897,30 +5897,30 @@ ITEM_20150729_005896
...
+ITEM_20150729_005908	The lost flag of an imperial corps. It is so severely damaged that it is impossible to know where it came from.
+ITEM_20150729_005909	An orb that records memories of the past
+ITEM_20150729_005910	A mysterious orb that reads the past from the user's own remembrance
+ITEM_20150729_005911	A keepsake emanating strong feelings of nostalgia
+ITEM_20150729_005912	A memento in which the deceased's lingering emotions and attachments are strongly felt. We don't know who he was, when he died, or what killed him.
+ITEM_20150729_005913	Ingredients needed to earn the Peltastas' cooperation 
+ITEM_20150729_005914	Blue Crystal
+ITEM_20150729_005915	Gravekeeper's Torch
+ITEM_20150729_005916	Torch received from Sigis. It is used to cremate bodies.
+ITEM_20150729_005917	Gravekeeper's voodoo  doll
```
Comment (Sunaries) [inline ITEM.tsv]: Extra spacing between voodoo and doll.

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**